Library/Homebrew/dev-cmd/linkage.rb

```diff
@@ -36,6 +36,8 @@ def linkage
             libgcc_s.so.1
             libgomp.so.1
             libstdc++.so.6
+
+            ld-linux-x86-64.so.2
```

Comment:
Please move this file to the top of the list (line 27). The files are grouped by the package that provides them, `glibc` and `gcc`.